docs/accordproject-concepts.md

```diff
@@ -113,19 +132,41 @@ _More information about Concerto can be found in the [Concerto Modeling](model-c
 
 While the combination of text and model already sets Accord Project template apart from a more traditional document format, the logic is what allows templates to really _come alive_. The logic allows a developer to associate _behavior_ to the template (specify what happens when a delivery is received late, check conditions for payment, automate interest rate calculations, determine if there has been a breach of contract, etc).
 
-The following screenshot shows the logic for the acceptance of delivery clause above.
-
-![Example Logic](/img/ergo-vscode.png)
+The following shows the logic for the **Acceptance of Delivery** clause above.
```

Comment:
Avoid "above"

Also, the paragraph before this should be "... and model already sets Accord Project templates apart from a more traditional ..."